### PR TITLE
Feature/trim get values

### PR DESCRIPTION
**Pull request by yuvalmarciano**
Fix comments from https://github.com/mkorman90/regipy/pull/245

### CODE REVIEW

File: regipy/plugins/system/bam.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def run(self):
                     version = None
                     entries = []
 
-                    for value in sid_subkey.get_values():
+                    for value in sid_subkey.get_values(trim_values=self.trim_values):
```
Comment (mkorman90) [inline regipy/plugins/system/bam.py:34]: You do not want to set `trim_values` as `True` for this specific plugin?

**Review by mkorman90 — CHANGES_REQUESTED** [review]

File: regipy/plugins/system/bam.py (hunk 1 above)
Comment (yuvalmarciano) [inline regipy/plugins/system/bam.py:34]:
```suggestion
                    for value in sid_subkey.get_values(trim_values=True):
```

Comment (yuvalmarciano) [reply regipy/plugins/system/bam.py]: Thanks!!

**Review by mkorman90 — APPROVED** [review]

**Comment by yuvalmarciano** [comment]
Thanks @mkorman90 ! Can we merge?

**Pull request merged**